### DISCUSSION TITLE
ESP32-4827S043 LVGL rotate screen problem

### DISCUSSION THREAD

**Discussion by The-Nicolini** [moononournation/Arduino_GFX#254] [General]
Hi as the title shows i am running esp32-4827s043 lovely screen i am using squareline in and lgvl
i have been trying everything .
I have everything running in landscape mode the original screen orientation w 480 h 272 then everything is perfect .
lucky for me thats the orientation i need today  but next week i need portred mode .

is this a known issue or am i doing something wrong (probaly) ??

i am using your latest library at the moment

**Comment by The-Nicolini** [comment]
[ui.txt](https://github.com/moononournation/Arduino_GFX/files/10461624/ui.txt)
config like this it fly,s how do i rotate it

**Reply by The-Nicolini** [reply]
off course love your work

**Reply by moononournation** [reply]
RGB display not support rotate in hardware. But you can config rotation in LVGL.
